site2/docs/schema-understand.md

```diff
@@ -220,7 +220,7 @@ This example shows how to construct a key/value schema and then use it to produc
 
 1. Construct a key/value schema with `INLINE` encoding type.
 
-    ```text
+    ```
```

Comment:
If the example is using java,  i recommend using  <code>``` java</code>  here, keywords can be highlighted

Reply:
Ok,I'll fix it.

Reply:
I'v fixed,PTAL.